sidecar/proxy/envoyadapter.go

```diff
@@ -17,11 +17,14 @@ package proxy
 import (
 	"sync"
 
+	"fmt"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/amalgam8/amalgam8/pkg/api"
 	"github.com/amalgam8/amalgam8/sidecar/config"
 	"github.com/amalgam8/amalgam8/sidecar/proxy/envoy"
 	"github.com/amalgam8/amalgam8/sidecar/proxy/monitor"
+	"github.com/amalgam8/amalgam8/sidecar/register"
 )
```

Comment:
Please rename this to discovery or something more meaningful. Register package is Envoy's SDS api right?